packages/amplify-provider-awscloudformation/src/aws-utils/aws-cfn.js

```diff
@@ -383,6 +385,39 @@ class CloudFormation {
           reject(err);
         }
       });
+      if (deleteS3) {
+        new S3(this.context, {}).then(s3 => {
+          const amplifyDir = this.context.amplify.pathManager.getAmplifyDirPath();
+          const tempDir = `${amplifyDir}-${envName}/.temp`;
```

Comment:
I think this tempDir should be created inside the amplify directory. Also please use path.join() instead of using `/` explicitly, as this would fail in windows.